src/web/entities/Header.jsx

```diff
@@ -11,45 +11,47 @@ import TableRow from 'web/components/table/Row';
 import PropTypes from 'web/utils/PropTypes';
 import SelectionType from 'web/utils/SelectionType';
 
-const defaultactions = (
+const defaultActions = (
   <TableHead align="center" title={_l('Actions')} width="8%" />
 );
 
 /**
  * A higher order component to create table headers which support entity
- * selection
+ * selection.
  *
- * If a react element instance is passed via actions the element will be
- * forwarded as actions to Component.
+ * If a React element instance is passed via actionsColumn, the element will be
+ * forwarded as actionsColumn to the Component.
  *
- * If actions is undefined a default table head column will be passed as
- * actions to Component.
+ * If actionsColumn is undefined, a default table head column will be passed as
+ * actionsColumn to the Component.
  *
- * If actions is true a default table head column will be passed as actions to
- * Component if the current selectionType (passed via props) is SELECTION_USER.
+ * If actionsColumn is true, a default table head column will be passed as actionsColumn
+ * to the Component if the current selectionType (passed via props) is SELECTION_USER.
  *
- * If actions is false no actions (a null value in react) will be passed to
- * Component.
+ * If actionsColumn is false, no actions (a null value in React) will be passed to
+ * the Component.
  *
- * @param {Element}   actions_column  React element, undefined or boolean value.
+ * @param {Element}   actionsColumn   React element, undefined or boolean value.
```

Comment:
I think this should be updated:

```suggestion
 * @param {Element | Boolean | undefined} actionsColumn React element, undefined or boolean value.
```

Reply:
i didn't update the docs of the other HOC :-/